server/routes/journeys/update-review/close-csip/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from 'express'
+import { PatchReviewController } from '../../base/patchReviewController'
+
+export class UpdateCloseCsipController extends PatchReviewController {
+  GET = async (req: Request, res: Response) => {
+    req.journeyData.isUpdate = true
+    res.render('record-review/close-csip/view', {
+      closeCsip: res.locals.formResponses?.['closeCsip'] ?? req.journeyData.review?.outcome,
```

Comment:
i don't think this is needed?

Reply:
It might be needed on the small chance we get an API error and they try again

Reply:
it's in the session. if they lost it, then they would have lost req.journeyData.csipRecord as well, and need to restart the journey from the beginning

Reply:
`req.journeyData.review?.outcome` would be "open" on a page refresh tho because the POST handler hasnt hit if there was an API error. So we bring the last entered value back from `formResponses`. Am I missing something?